pages/levels/5.vue

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+
+const code = `    `;
+
+export default defineComponent({
+  data: () => {
+    return {
+      markdown: `What’s the deal with POSH? POSH stands for \'Plain Old Semantix HTML\'. It basically means you should use the right HTML elements for their intended purpose, like using a button instead of a div with a click function.
```

Comment:
"it basically means you shoudl use the .... intended purpose". statt was? was sollte man sonst nutzen? das steht nicht hier und ist verwirrend, wenn man sich nicht auskennt.

Reply:
es steht man soll einen button statt div mit click nutzen, was meinst du?

Reply:
Naja weil die "click function" hat ja der button dann noch immer, also das ist ja kein Alleinstellungsmerkmal des `div`s